packages/stone-phaser/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+pkgname=stone-phaser
+pkgver=0.1.2
+pkgrel=1
```

Comment:
I think this hinges on a general decision on how we treat the AUR. What actually happens, if the AUR package gets bumped (either pkgrel or version) after the user install our version? (unlikely in this case, since I maintain the AUR package too.)

How would we treat packages where we don't have control over the AUR counterpart?

Reply:
Most AUR helpers will just pick the highest version number and if repo > AUR, install the repo version.
However if we have a AUR > repo, it's nothing we can control, but since updating an AUR package (again, using an AUR helper like yay) requries manual confirmation and takes longer, most users may not choose to do that.

Anyway this only affects users updating their system with `yay -Syu` and they should know what they're doing.

But let's do some research eventually how other binary repos are dealing with that.

Reply:
Bumped pkgrel in ed5b781.